File: Armstrong.java
import java.util.Scanner;
public class Armstrong{
  public static void main(String[] args){
  Scanner input = new Scanner(System.in);

  System.out.print("Enter Factorial Number: ");
  int num = input.nextInt();


remainder = 0;
result = 0;
int number = num;

while(number != 0){
remainder = number % 10;
result = math.pow(remainder,3);
number /= 10
}
if(result == number){
System.out.println(num + " is an Armstrong number ")
}

else{
System.out.println(num + "is not an Armstrong number")
}

}

}

puclic static boolean Armstrong(int num){
if(num < 0){
return false;
}

if(num >= 0 && num <= 9){
return true;

}
int number = num;
int sum = 0;


int numbers = String.value(num).length():

for int i = num; i > 0; i /= 10){

int digit = i % 10;
sum += Math.pow(digit,numbers);

}
return sum;
}

File: Armstrong.js
 function checkArmstrong() {
  const prompt = require("Enter a number")(); 
  let num = parseInt(prompt("Enter a number: "));

  let remainder = 0;
  let result = 0;
  let originalNumber = num;

  while (num > 0) {
    remainder = num % 10;
    result += Math.pow(remainder, 3); 
    num = Math.floor(num / 10); 
  }

  if (result === originalNumber) { 
    console.log(originalNumber + " is an Armstrong number.");
  } else {
    console.log(originalNumber + " is not an Armstrong number.");
  }
}




function Armstrong(num) {
  if (num < 0) {
    return false;
  }
  if (num >= 0 && num <= 9) {
    return true;
  }

  let number = num;
  let sum = 0;
  let numbers = String(num).length; 


  while (number > 0) {
    let digit = number % 10;
    sum += Math.pow(digit, numbers);
    number = Math.floor(number / 10);
  }

   return sum === num; 
}

